blocks/blockoracle_test.go

```diff
@@ -182,7 +182,7 @@ func TestBlockOracleEmptyActiveSet(t *testing.T) {
 	blockOracle := NewMinerBlockOracle(committeeSize, layersPerEpoch, activationDB, beaconProvider, vrfsgn, nodeID, func() bool { return true }, lg.WithName("blockOracle"))
 
 	_, proofs, _, err := blockOracle.BlockEligible(types.NewLayerID(layersPerEpoch * 3))
-	r.EqualError(err, "zero total weight not allowed")
+	r.EqualError(err, "oracle get num slots: zero total weight not allowed")
```

Comment:
would be nicer to use ErrorIs for this checks

Reply:
done